scripts/Muon/GUI/Common/contexts/muon_context.py

```diff
@@ -13,15 +13,19 @@
 from Muon.GUI.Common.contexts.muon_data_context import MuonDataContext
 from Muon.GUI.Common.contexts.muon_group_pair_context import MuonGroupPairContext
 from Muon.GUI.Common.contexts.muon_gui_context import MuonGuiContext
+from Muon.GUI.Common.contexts.phase_table_context import PhaseTableContext
 from Muon.GUI.Common.utilities.run_string_utils import run_list_to_string
+import Muon.GUI.Common.ADSHandler.workspace_naming as wsName
+from Muon.GUI.Common.contexts.muon_data_context import get_default_grouping
 
 
 class MuonContext(object):
     def __init__(self, muon_data_context=MuonDataContext(), muon_gui_context=MuonGuiContext(),
-                 muon_group_context=MuonGroupPairContext(), base_directory='Muon Data'):
+                 muon_group_context=MuonGroupPairContext(), base_directory='Muon Data', muon_phase_context= PhaseTableContext()):
         self._data_context = muon_data_context
         self._gui_context = muon_gui_context
         self._group_pair_context = muon_group_context
+        self.phase_context = muon_phase_context
```

Comment:
should this be `self._phase_context` to be consistent with the others?

Reply:
Yes I think so